crates/matrix-sdk-ui/src/timeline/mod.rs

```diff
@@ -56,6 +56,7 @@ mod futures;
 mod inner;
 mod item;
 mod pagination;
+pub mod polls;
```

Comment:
Can you please make this module private like the other ones and re-export any public types from this module (as part of the `pub use` statement starting on line 71)?

Reply:
Sure I will, can you offer some background on the why to let me learn something new?